app/components/templates/templatesController.js

```diff
@@ -84,7 +84,8 @@ function ($scope, $q, $state, $filter, $anchorScroll, Config, Info, Container, C
         PortBindings: {},
         Binds: [],
         NetworkMode: $scope.formValues.network.Name,
-        Privileged: false
+        Privileged: false,
```

Comment:
Could you remove the changes to the TemplatesController ? It should not be affected by this PR.